tests/test_parser/test_openapi.py

```diff
@@ -481,33 +493,37 @@ def test_add_parameters_handles_no_params(self):
 
         endpoint = self.make_endpoint()
         schemas = Schemas()
+        parameters = Parameters()
         config = MagicMock()
 
         # Just checking there's no exception here
         assert Endpoint.add_parameters(
-            endpoint=endpoint, data=oai.Operation.construct(), schemas=schemas, config=config
-        ) == (
-            endpoint,
-            schemas,
-        )
+            endpoint=endpoint, data=oai.Operation.construct(), schemas=schemas, parameters=parameters, config=config
+        ) == (endpoint, schemas, parameters)
 
     def test_add_parameters_parse_error(self, mocker):
         from openapi_python_client.parser.openapi import Endpoint
 
         endpoint = self.make_endpoint()
         initial_schemas = mocker.MagicMock()
+        initial_parameters = mocker.MagicMock()
```

Comment:
Oof, sorry about the complexity of some of these tests. I've been trying to move away from mocking and toward more end-to-end (put in schema, get out result or error) tests but it's a lot of work to rewrite and get coverage 😓

Reply:
All good. I was able to figure it out.